app/javascript/components/Download/RichContent.tsx

```diff
@@ -277,28 +279,31 @@ const FileEmbedGroupNodeView = ({ node }: NodeViewProps) => {
 
   return (
     <NodeViewWrapper>
-      <div role="tree" ref={ref}>
-        <div role="treeitem" aria-expanded={expanded}>
-          <div className="content" onClick={() => setExpanded(!expanded)} contentEditable={false}>
+      <Rows role="tree" ref={ref}>
+        <Row role="treeitem" aria-expanded={expanded}>
+          <RowContent onClick={() => setExpanded(!expanded)} contentEditable={false}>
+            <Icon name={expanded ? "outline-cheveron-down" : "outline-cheveron-right"} />
```

Comment:
This is very simple and not used in a huge number of places, so I don't think it's worth a separate component (similar to table actions columns where we moved the flexbox styling inline instead of creating a component/prop)

The same goes for the rest of the expand/collapse logic; it doesn't really simplify things much to add it to the rows component

We should consider merging/refactoring some of these components in general though, the rich text editor in particular is super hard to follow